src/main/java/remi/model/Deadline.java

```diff
@@ -0,0 +1,34 @@
+package remi.model;
+
+/**
+ * Represents a deadline task that ends by a certain time. Subclass of Task.
+ */
+public class Deadline extends Task{
+    private String by;
```

Comment:
Perhaps it is better to give the due date variable a meaningful and self-explaining name? Something like `dueDate`.